apps/web/src/learning/discussion-editor/atomic-editor.css

```
@@ -0,0 +1,56 @@
+/* Atomic/CodeMirror owns these internals, so the feature-local overrides below
+ * cannot be expressed reliably with Tailwind utilities on React-owned nodes. */
+.learning-discussion-atomic-editor {
+  --atomic-editor-bg: transparent;
+  --atomic-editor-fg: var(--text);
+  --atomic-editor-muted: var(--muted);
+  --atomic-editor-link: var(--accent-ink, var(--accent));
+  --atomic-editor-measure: none;
+  --atomic-editor-font: inherit;
+  --atomic-editor-body-size: 0.9375rem;
+  --atomic-editor-body-leading: 1.6;
+  height: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)